experimental/waveshare_eink/linux_kernel_modules/fb_waveshare_eink.c

```diff
@@ -72,6 +72,16 @@ const u8 lut_partial_update[] = {
 	0x00, 0x00, 0x00, 0x00, 0x00, 0x00
 };
 
+static u8 bitrev8(u8 byte) 
```

Comment:
`bitrev8` is a standard linux function.  See [`here`](https://elixir.bootlin.com/linux/v3.18.136/source/include/linux/bitrev.h#L8).  Please `#include <linux/bitrev.h>` and use that version.